README.md

```diff
@@ -122,7 +122,7 @@ Now securely transfer the `meetingResponse` and `attendeeResponse` objects to yo
 These objects contain all the information needed for a client application using the Amazon Chime SDK for JavaScript to join the meeting.
 
 The value of the MediaRegion parameter in the createMeeting() should ideally be set to the one of the media regions which is closest to the user creating a meeting.
-An implementation can be found in the [getNearestMediaRegion()](https://github.com/aws/amazon-chime-sdk-js/blob/master/demos/browser/app/meetingV2/meetingV2.ts#L460)
+An implementation can be found in the [getNearestMediaRegion()](https://github.com/aws/amazon-chime-sdk-js/blob/master/demos/browser/app/meetingV2/meetingV2.ts#L531)
```

Comment:
Actually the meetingV2 demo will keep growing and the line number will change accordingly. If you check the latest demo code, [meetingV2#L548](https://github.com/aws/amazon-chime-sdk-js/blob/0ba017f4c3d3ef418676bff667b39a9e75cea0fc/demos/browser/app/meetingV2/meetingV2.ts#L548) points to that function and not line no. 531.

I think it would be more appropriate and stable if we just callout to check `getNearestMediaRegion()` in the [meetingV2](https://github.com/aws/amazon-chime-sdk-js/blob/master/demos/browser/app/meetingV2/meetingV2.ts) browser demo app.

Reply:
Yeah, I agree...
'getNearestMediaRegion() in the meetingV2 browser demo app.` sounds good to me
Or we can point to a public doc like https://docs.aws.amazon.com/chime/latest/dg/chime-sdk-meetings-regions.html